devtools/src/arcs-pec-log.js

```diff
@@ -196,7 +196,7 @@ class ArcsPecLog extends MessengerMixin(PolymerElement) {
     if (newFilteredEntries.length) {
       this.push('filteredEntries', ...newFilteredEntries);
     }
-    this.downloadEnabled = !!this.filteredEntries.length;
+    this.downloadEnabled = Boolean(this.filteredEntries.length);
```

Comment:
use `(this.filteredEntries.length !== 0)` instead (and for all other uses of Boolean too)